src/Sentry/SentryEvent.cs

```diff
@@ -197,6 +197,8 @@ internal bool HasTerminalException()
         ) ?? false;
     }
 
+    internal DynamicSamplingContext? DynamicSamplingContext { get; set; }
```

Comment:
This means it doesn't work with offline mode? That's OK btw, just checking.

Reply:
It still works.  We cache envelopes, including the headers - so that includes the `trace` item header where the DSC is written.